### PR TITLE
Allow changing topics dinamically

### PR DESCRIPTION
**Pull request by v0idpwn**
Required adding a public API module. Called it `BroadwayKafka`.

### CODE REVIEW

**Comment by josevalim** [comment]
It looks good! We just need some docs and examples!

**Comment by v0idpwn** [comment]
I didn't manage to run the integration tests here, and the CI won't do it too, so I couldn't write an integration test. Still, I manually tested it and it worked. Maybe the CI could run the integration tests? I would be happy to add this functionality if needed.

File: lib/broadway_kafka.ex [hunk 1]
```diff
@@ -0,0 +1,30 @@
+defmodule BroadwayKafka do
+  @moduledoc """
+  Helpers for BroadwayKafka
+
+  For the producer, view `BroadwayKafka.Producer`
+  """
```
Comment (v0idpwn) [inline lib/broadway_kafka.ex:6]: Ain't sure about what to write here :')

**Comment by josevalim** [comment]
If there is an easy way to get Kafka running on GitHub CI, that would be welcome!

**Comment by josevalim** [comment]
:green_heart: :blue_heart: :purple_heart: :yellow_heart: :heart:

**Pull request merged**